config/express.js

```diff
@@ -28,55 +28,50 @@ module.exports = (app, config) => {
     });
   log.info({ config: { nunjucksEnvironment } }, 'nunjucks environment configuration');
 
-  app.use(helmet.contentSecurityPolicy({
-    directives: {
-      defaultSrc: [
-        '\'self\'',
-      ],
-      childSrc: [
-        'https://*.hotjar.com:*',
-      ],
-      scriptSrc: [
-        '\'self\'',
-        '\'unsafe-inline\'',
-        '\'unsafe-eval\'',
-        'data:',
-        'www.google-analytics.com',
-        's.webtrends.com',
-        'statse.webtrendslive.com',
-        'static.hotjar.com',
-        'script.hotjar.com',
-        'cdn.jsdelivr.net',
```

Comment:
is `cdn.jsdelivr.net` no longer used?

Reply:
I'm not sure it was ever used and if it was there are no requests from any of the apps going to that domain now.